appuio/cloud-portal/templates/configmap.yaml

```
@@ -1,9 +1,23 @@
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "cloud-portal.fullname" . }}
+  name: {{ include "cloud-portal.fullname" . }}-frontend
   labels:
     {{- include "cloud-portal.labels" . | nindent 4 }}
 data:
   config.json: |
     {{- toJson .Values.portal.config | nindent 4 }}
+{{- if .Values.portal.env }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "cloud-portal.fullname" . }}-env
+  labels:
+    {{- include "cloud-portal.labels" . | nindent 4 }}
+data:
+  {{- range $key, $value := .Values.portal.env }}
+  {{ $key }}: {{ $value }}
+  {{- end }}
```

```suggestion
{{ toYaml .Values.portal.env | indent 2 }}
```

Could be a bit safer.